src/commands/BootstrapCommand.js

```diff
@@ -346,7 +346,9 @@ export default class BootstrapCommand extends Command {
         async.series(root.map(({name, dependents}) => (cb) => {
           async.series(dependents.map(({nodeModulesLocation: dir}) => (cb) => {
             if (dir === this.repository.nodeModulesLocation) return cb();
-            FileSystemUtilities.rimraf(path.join(dir, name), cb);
+            const filePath = path.join(dir, name);
+            if (!FileSystemUtilities.existsSync(filePath)) return cb();
+            FileSystemUtilities.rimraf(filePath, cb);
```

Comment:
It's not that it _fails_, it's just _slow_ compared to calling `rimraf` in-process.